importers/yelp.com/download_html.py

```diff
@@ -55,14 +58,14 @@ def download_location_detail_page(url):
 	name = get_location_name_from_url(url)
 	output_file_name = 'data/raw_html_'+ name +'.html'
 	if not os.path.exists(output_file_name):
-		print ('Downloading details for location: ' + name)
-		response = urllib2.urlopen(url)
+		print(('Downloading details for location: ' + name))
+		response = urllib.request.urlopen(url)
 		html = response.read() # returns all the lines in a file.
-		with open(output_file_name, 'w') as html_file:
+		with open(output_file_name, 'wb') as html_file:
 			html_file.write(html)
 			html_file.close()
 	else:
-		print ('Already have details for location: ' + name)
+		print(('Already have details for location: ' + name))
```

Comment:
I do not know, I just use the command 2to3 to migrate the code from python 2 to python 3, then it can work in Python3, however, I did not check this line, and I did not know it added double brackets.

Reply:
I think it is a little bug of the tool.  The tool might have assumed there would be no brackets for the print statement's expression since the input was Python 2.7.  This assumption is invalid as you can see.

Remove the extra brackets here and elsewhere in the file so the code looks better and give it a test run.
Something like this should work: ```print('Already have details for location: ' + name)```